.github/workflows/release_to_pypi.yml

```diff
@@ -33,6 +33,11 @@ jobs:
           pip install --compile --no-cache-dir pycurl
           pip install -U .[test]
 
+      - name: Pre Commit Checks
```

Comment:
this shouldn't be used anymore, its deprecated

instead pre-commit.ci should be used